src/queue.ts

```diff
@@ -31,7 +32,7 @@ export class Queue<TItem> extends EventEmitter {
     }).promise()
   }
 
-  startProcessing (handler : (item : TItem) => any | PromiseLike<any>, options?: ProcessOptions): PromiseLike<void> {
+  startProcessing (handler : (item : TItem, message : any) => any | PromiseLike<any>, options?: ProcessOptions): PromiseLike<void> {
```

Comment:
Another thing, the notation of the handler should be `(item : TItem, message? : any) => any | PromiseLike<any>`, to indicate the `message` parameter is optional.